packages/firestore/test/unit/remote/node/serializer.test.ts

```diff
@@ -25,6 +25,9 @@ import { GeoPoint } from '../../../../src/api/geo_point';
 import { Timestamp } from '../../../../src/api/timestamp';
```

Comment:
I couldn't figure out how to test `KeyFieldFilter` due to problems with creating a query with this filter.

Reply:
Not sure if this is what you mean, but it might look something like:

```
      const input = filter(DOCUMENT_KEY_NAME, '==', ref('project/database', 'coll/doc'));
```

Reply:
Thanks! That worked.